mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2370,6 +2370,26 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+{{ if ne $.TargetVersionName `ga` -}}
+			"pod_autoscaling": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: `The config for pod autoscaling.`,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"hpa_profile": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
```

Comment:
just want to double-check that the API returns a complex default? Optional + Computed has some drawbacks, documented at https://googlecloudplatform.github.io/magic-modules/develop/diffs/.